README.md

```diff
@@ -6,68 +6,47 @@
-    mobile and web should be supported so that both the user and the developer
-    feel comfortable on their platform of choice.
-
-## The .slint Markup Language
-
-Slint comes with a markup language that is specifically designed for user
-interfaces. This language is easy to learn, to read and write, and provides
-a powerful way to describe graphical elements, their placement, and the flow of
-data through the different states. It's a familiar syntax to describe the
-hierarchy of elements and property bindings.
-
-Here's the obligatory "Hello World":
-
-```slint
-export component HelloWorld inherits Window {
-    width: 400px;
-    height: 400px;
-
-    Text {
-       y: parent.width / 2;
-       x: parent.x + 200px;
-       text: "Hello, world";
-       color: blue;
-    }
-}
-```
+Slint is a declarative GUI toolkit to build native user interfaces for desktop
+and embedded applications written in Rust, C++, or JavaScript. The name *Slint*
+is derived from our design goals:
+
+- **Scalable**: Slint should support responsive UI design, allow cross-platform
+    usage across operating systems and processor architectures and support
+    multiple programming languages.
+- **Lightweight**: Slint should require minimal resources, in terms of memory
+    and processing power, and yet deliver a smooth, smartphone-like user
+    experience on any device.
+- **Intuitive**: Designers and developers should feel productive while enjoying
+    the GUI design and development process. The design creation tools should be
+    intuitive to use for the designers. Similarly for the developers, the APIs
+    should be consistent and easy to use, no matter which programming language
+    they choose.
+- **Native**: GUI built with Slint should match the end users' expectations of a
+    native application irrespective of the platform - desktop, mobile, web or
+    embedded system. The UI design should be compiled to machine code and provide
+    flexibility that only a native application can offer: Access full operating
+    system APIs, utilize all CPU and GPU cores, connect to any peripheral.
+
+We invite you to use Slint and be part of its community.
 
 ## Current Status
 
-Slint is in active development. The state of the toolkit for each platform is as
+Slint is in active development. The state of support for each platform is as
 follows:
 
-- **Embedded**: *Ready* Slint is already being used in production projects by
-    customers (running on an Arm processor with Linux).
-- **Microcontrollers** (MCU): *Ready*. Slint apps can run on MCUs with less than
-    300K of RAM.
+- **Embedded**: *Ready* Slint runtime requires less than 300KiB of RAM. Slint is
+    being used by customers in production projects on embedded devices running
+    embedded Linux and Windows. Slint can run on different processor architectures
+    such as ARM Cortex M, ESP32, STM32 from the MCU category to ARM Cortex A,
+    Intel x86 from the MPU category.
```

Comment:
Rephrasing to put emphasis on being used by customers in production. The fact that memory usage can be low is something we can easily mention in the second sentence, but use by customers weights more IMO.

```suggestion
- **Embedded**: *Ready* Slint is being used by customers in production on embedded
    devices running embedded Linux and Windows. The Slint run-time requires less than
    300KiB of RAM and can run on different processor architectures such as ARM Cortex M,
    ESP32, STM32 from the MCU category to ARM Cortex A, Intel x86 from the MPU category.
```